src/tests/ftest/osa/offline_reintegration.py

```diff
@@ -166,7 +162,7 @@ def run_offline_reintegration_test(self, num_pool, data=False,
                         continue
                     output = self.dmg_command.pool_reintegrate(self.pool.uuid,
                                                                rank[val])
-                self.print_and_assert_on_rebuild_failure(output)
+                self.print_and_assert_on_rebuild_failure(output, timeout=15)
```

Comment:
This 15 will eventually be passed into `TestPool.wait_for_rebuild()` as an interval. I wonder why changing the interval from the default 3 to 15 will make a difference.

Reply:
It will give opportunity for DAOS rebuild code to take more action. Based on wangdi comment on the ticket "cont_rf_check_ult(), which restores the RF(permission) after reintegration, did not step in yet"... I am thinking increasing the default value will help some of the rebuild code ULTs to complete their task.